tests/.sauce/config_on_sauce_mac.yml

```
@@ -0,0 +1,15 @@
+apiVersion: v1alpha
+kind: puppeteer-replay
+sauce:
+  region: us-west-1
+  concurrency: 10
+suites:
+  - name: "getting some coffee"
+    platform: "macOS 11.00"
```

Maybe we should set ourselves up with a good ol' leading by example and use the newest version of mac 💪 
```suggestion
    platform: "macOS 12"
```